umbraco/Umbraco-CMS
# Added map to api attribute

Added attribute to map APIs to a given open api definition (e.g. management API vs delivery api)

### Test
-  Create another definition using  `swaggerGenOptions.SwaggerDoc(...)`, with other name e.g. "delivery-v1"
   - Maybe you will need to map a swagger endpoint too using `swaggerUiOptions.SwaggerEndpoint(...)`
- Ensure the definition does not include any management apis.
